src/planscape/planscape/settings.py

```diff
@@ -178,3 +178,8 @@
 ACCOUNT_EMAIL_VERIFICATION = 'none'
 ACCOUNT_LOGOUT_ON_GET = True
 
+# PostGIS values
```

Comment:
is there any documentation on where this comes from/how it's used?

Reply:
I'll beef this up.

Reply:
Done.